apps/coscrad-frontend/src/utils/test-utils/renderWithProviders.tsx

```diff
@@ -2,10 +2,11 @@ import { PreloadedState } from '@reduxjs/toolkit';
 import { render, RenderOptions } from '@testing-library/react';
 import { PropsWithChildren } from 'react';
 import { Provider } from 'react-redux';
-import { AppStore, RootState, setupStore } from '../store';
+import { AppStore, RootState, setupStore } from '../../store';
 interface ExtendedRenderOptions extends Omit<RenderOptions, 'queries'> {
     preloadedState?: PreloadedState<RootState>;
     store?: AppStore;
+    route?: string;
```

Comment:
Is there room in renderWithProviders for injecting a dummy config?

Reply:
I mentioned that possibility in the story for creating a content config provider.